src/sentry/features/temporary.py

```diff
@@ -186,6 +186,7 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:profiling-global-suspect-functions", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     manager.add("organizations:profiling-summary-redesign", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     manager.add("organizations:profiling-using-transactions", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+    manager.add("organizations:continuous-profiling", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
There is a chance of flakiness when you use Flagr for a feature in relay project configs: If Flagr times out, the feature defaults to "disabled". Because project configs are cached for up to one hour, this might accidentally disable the feature.

TL;DR if you need the feature to be enabled consistently, use `FeatureHandlerStrategy.INTERNAL`.